index.html

```diff
@@ -1776,8 +1784,11 @@ <h2 id="OfflineAudioCompletionEvent">
             void disconnect()
           </dt>
           <dd>
-            <p>Disconnects a single output of the <a><code>AudioNode</code></a> from any other <a><code>AudioNode</code></a>
-            or <a><code>AudioParam</code></a> objects to which it is connected.</p>
+            <p>
+              Disconnects a single output of the <a><code>AudioNode</code></a>
```

Comment:
Is this change because tidy wanted to wrap these lines and it didn't before?  (There are quite a few of these below.)